app/lib/index.js

```diff
@@ -221,6 +216,34 @@ renderer.on('config:set', function(key, value, ...args) {
   }
 });
 
+// error tracking //////////
+renderer.on('sentry:initialize', function(params) {
+  const { dsn, releaseTag, editorID } = params;
+
+  try {
+
+    // we tag the release with -backend postfix as we don't want to
+    // use the same tag with frontend Sentry in order not to have problems
+    // with source maps.
+    Sentry.init({ dsn, release: releaseTag + '-backend' });
```

Comment:
Since we upload sourcemaps tagged with "releaseTag" variable, we add -backend prefix here in order to inform Sentry that error coming from this Sentry instance has got nothing to do with the sourcemap we uploaded via frontend webpack.

Reply:
But we stil want to tag the backend Sentry instance too, in order to see which version caused the error.

Reply:
> Since we upload source maps tagged with "releaseTag" variable, we add -backend prefix here in order to inform Sentry that error coming from this Sentry instance has got nothing to do with the source map we uploaded via frontend webpack.

Is this the sentry electron best practice? Any resources you found?

Reply:
Sentry offers an API for Electron which I did not initially use as we depended on Privacy Prefernces checks (in Frontend).

I might integrate this library now as we are ok with doing Privacy Preferences checks in backend.

Reply:
That sounds like a great idea :+1: